pom.xml

```diff
@@ -61,6 +61,7 @@
     <!-- Ensure older versions of spring still work -->
     <spring4.version>4.3.13.RELEASE</spring4.version>
     <spring.version>3.2.18.RELEASE</spring.version>
+    <spring-amqp.version>1.7.5.RELEASE</spring-amqp.version>
```

Comment:
until used in multiple modules, or pinned to a specific spring version, keep defined as a property in the instrumentation module itself. The reason spring is listed here is because multiple dependents on core spring